pokemongo_bot/cell_workers/spin_fort.py

```diff
@@ -27,9 +27,13 @@ def __init__(self, bot, config):
         super(SpinFort, self).__init__(bot, config)
 
     def initialize(self):
+        # 10 seconds from current time
+        self.next_update = datetime.now() + timedelta(0, 10)
+
         self.ignore_item_count = self.config.get("ignore_item_count", False)
         self.spin_wait_min = self.config.get("spin_wait_min", 2)
         self.spin_wait_max = self.config.get("spin_wait_max", 3)
+        self.min_interval = int(self.config.get('min_interval', 120))
```

Comment:
Throughout the code, the 'min_interval' is used to show some logging/stats updates. This is the reason why I decided to keep this name since its purpose is to show a warning the the spin limit has been reached. 

I have added some documentation for this variable as well.